app/components/ui/tale-tab-metadata/component.js

```diff
@@ -0,0 +1,91 @@
+import Ember from 'ember';
+import { later } from '@ember/runloop';
+import { computed } from '@ember/object';
+import { not } from '@ember/object/computed';
+
+import $ from 'jquery';
+
+import config from '../../../config/environment';
+
+const taleStatus = Object.create({
+  NONE: -1,
+  READ: 0,
+  WRITE: 1,
+  ADMIN: 2,
+  SITE_ADMIN: 100
+});
+
+export default Ember.Component.extend({
+  apiHost: config.apiHost,
+  environments: [],
+  model: null,
+  init() {
+    this._super(...arguments);
+    
+    // Fetch images for user to select an environment
+    let component = this;
+    $.getJSON(this.get('apiHost') + '/api/v1/image/').then(function(images) {
+      component.set('environments', images);
+      component.selectDefaultImageId();
+    });
+  },
+  
+  didRender() {
+    // Enable the environment dropdown functionality
+    $('.ui.dropdown').dropdown();
+    this.selectDefaultImageId();
+  },
+  
+  selectDefaultImageId() {
+    // Select the current imageId by default
+    let selectedImageId = this.get("model").get("tale").get("imageId")
+    $('.ui.dropdown').dropdown('set selected', selectedImageId);
+  },
+  
+  canEditTale: computed('model.tale._accessLevel', function () {
+    return this.get('model') && this.get('model').get('tale') && this.get('model').get('tale').get('_accessLevel') >= taleStatus.WRITE;
+  }).readOnly(),
+  cannotEditTale: computed.not('canEditTale').readOnly(),
+  
+  actions: {
+    // TODO: Status messages are still missing
+    updateTale() {
+      let tale = this.get("model").get("tale");
+      let component = this;
+      //component.set("tale_creating", true);
+      console.log("Sending updated tale:", tale);
+      console.log("New image ID:", tale.imageId);
+      
+      let onSuccess = (item) => {
+      //component.set("tale_creating", false);
+      //component.set("tale_created", true);
```

Comment:
We can probably get rid of these comments since tale_created, tale_creating, tale_not_created, and error_msg aren't used here or in the template.

Reply:
Understood! I wasn't sure if the long-term plan was to move them over or to just get rid of them, but I can at least remove them for now